extensions/azurecore/src/azureResource/providers/databaseServer/databaseServerService.ts

```diff
@@ -14,20 +17,57 @@ export interface DbServerGraphData extends GraphData {
 	};
 }
 
-export const serversQuery = `where type == "${azureResource.AzureResourceType.sqlServer}"`;
+/**
+ * Properties returned by the Synapse query are different from the server ones and have to be treated differently.
```

Comment:
I'd suggest instead adding separate doc comments on each of the fields below explaining what they are. e.g.

```typescript
/**
 * Resource group used for any SQL pools inside the workspace, which may be different than the resource group of the workspace itself
 */
 managedResourceGroupName: string;
 ```